### DISCUSSION TITLE
Is there a way to write integers sequentially per cursor?

### DISCUSSION THREAD

**Discussion by emummel20** [helix-editor/helix#8108] [Q&A] [answered]
Title is stupid sorry but couldn't think of a better one. Real quick, if this is my cursor `[]`, and I use multi cursor so I have cursors in various locations

```
[]
[]

[]

[]
[]
[]

[]
```

and I wanted to produce an integer, sequentially, let's say from 1..N at each cursor location such that I end up with

```
[1]
[2]

[3]

[4]
[5]
[6]

[7]
```

is there a way to do that?

**Comment by the-mikedavis** [comment, marked as answer]
You can use the selection index register `#`: `"#p` will paste `1` for the first selection, `2` for the second and so on. You can also use it to increment each selection by their index, so `"#<C-a>` will increment the first selection by 1, the second by 2, etc..

**Reply by emummel20** [reply]
thanks
